README.md

```diff
@@ -32,7 +32,7 @@ an bitvector input, `'x'` with 3 bits becomes inputs `'x[0]', 'x[1]', 'x[3]'`
 
 # Installation
 
-If you just need to use `aiger_bv`, you can just run:
+If you just need to use `aigerbv`, you can just run:
```

Comment:
Was this an intentional change? The current module is `aiger_bv`

Reply:
Further down in the README 'import aigerbv' is used. I quickly tried both and on my machine only  the variant aigerbv worked.

Reply:
Import aiger_bv only works when you happen to be in the root directory of the package or repo, when it matches the directory name.

Reply:
@MarkusRabe I have a strong suspicion you have an old version of the package installed.

I would recommend doing:
`pip show py-aiger-bv`

or 

`pip freeze | grep py-aiger-bv`

and making sure you have `2.0.0` installed.

Reply:
Yes, turned out I had an old version. I'll update the PR shortly. Made the naming consistently aiger_bv throughout the README.